www/src/components/account/billing/BillingConsumptionProvider.tsx

```diff
@@ -0,0 +1,41 @@
+import { ReactNode, useContext, useMemo } from 'react'
+import { useQuery } from '@apollo/client'
+
+import CurrentUserContext from '../../../contexts/CurrentUserContext'
+import BillingConsumptionContext, { BillingConsumptionContextType } from '../../../contexts/BillingConsumptionContext'
+
+import { USERS_QUERY } from './queries'
+
+import BillingError from './BillingError'
+import BillingLoading from './BillingLoading'
+
+type BillingConsumptionProviderPropsType = {
+  children: ReactNode
+}
+
+function BillingConsumptionProvider({ children }: BillingConsumptionProviderPropsType) {
+  const { me } = useContext(CurrentUserContext)
+  const { data, loading, error } = useQuery(USERS_QUERY)
+
+  const nClusters = useMemo(() => (me?.account?.clusterCount ? parseInt(me?.account?.clusterCount) : 0), [me])
+  const nUsers = useMemo(() => data?.users?.edges?.length ?? 0, [data])
```

Comment:
we should **not** use this to count users, there's a user count on the account object as well.

Also it might be more consistent to fetch the account explicitly rather than sideload it via the user since that's technically cached server side.